From bd173ac44cb4ccf86e68cb8c2f85cd5b09a245b4 Mon Sep 17 00:00:00 2001
From: Greg Campbell <gtcampbell@gmail.com>
Date: Mon, 12 Apr 2010 23:50:26 -0700
Subject: [PATCH] Explicitly require core_ext/file/path

We're no longer using the shim, so this will need to be required to test
its functionality.
---
 activesupport/test/core_ext/file_test.rb |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/activesupport/test/core_ext/file_test.rb b/activesupport/test/core_ext/file_test.rb
index e1258b8..16b0bd3 100644
--- a/activesupport/test/core_ext/file_test.rb
+++ b/activesupport/test/core_ext/file_test.rb
@@ -1,5 +1,6 @@
 require 'abstract_unit'
 require 'active_support/core_ext/file'
+require 'active_support/core_ext/file/path'
 
 class AtomicWriteTest < Test::Unit::TestCase
   def test_atomic_write_without_errors
-- 
1.6.3.3

